matrix_sdk_base/src/client.rs

```diff
@@ -1424,6 +1427,11 @@ impl BaseClient {
                 event_emitter.on_room_power_levels(room, &power).await
             }
             RoomEvent::RoomTombstone(tomb) => event_emitter.on_room_tombstone(room, &tomb).await,
+            RoomEvent::CustomRoom(custom) => {
+                if let Ok(raw) = serde_json::value::to_raw_value(custom) {
```

Comment:
I assume we're converting the events here to raw json values so we avoid having two event emitter methods for custom events, correct?

If so I think we could get the same win by adding a enum for custom events, we're losing quite a bit of information by converting this to a raw json value and users need to check for all the fields instead of just checking the content.